examples/gno.land/p/demo/urequire/urequire.gno

```diff
@@ -0,0 +1,87 @@
+// urequire is a sister package for uassert.
+// XXX: codegen the package.
+package urequire
+
+import "gno.land/p/demo/uassert"
+
+// type TestingT = uassert.TestingT // XXX: bug, should work
```

Comment:
What is exactly the bug here? do we have an issue that we can add here to track it somehow?

Reply:
I remember seeing this _exact_ issue a while back in local dev, but forgot to open up an issue for it 🤦‍♂️